packages/terra-clinical-result/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ ChangeLog
 
 Unreleased
 -----------------
+### Fixed
+* Fixed an alignment issue with FlowsheetResultCell that caused normal interpretation results to be aligned to the left.
```

Comment:
which change affected the alignment issue? Is this from trimming the interpretation value?

Reply:
For the alignment to be displayed correctly, we add a class when there is no icon which places a margin to line up the text as if there was an icon. We were mistaking the "normal" interpretation to be one that provided an icon and thus weren't providing that class. The inclusion of [this array](https://github.com/cerner/terra-clinical/pull/621/files#diff-a81e0ffc96673dad2a43e3b9b81f7090R63) and the changes to see if an interpretation are in that array are how this PR fixes that issue.

Reply:
Thanks for the info!

Reply:
No. This adds extra space on the left to any value which doesn't have an interpretation that also displays an icon, so that all values, icon or without icon, all vertically align when in a large flowsheet grid.